Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTriggerTest.kt

```diff
@@ -102,24 +102,6 @@ internal class DccWalletInfoUpdateTriggerTest : BaseTest() {
         }
     }
 
-    @Test
-    fun `update is not triggered on same certificates change`() = runTest2(true) {
```

Comment:
Would it make sense to keep this test as it tests something else? I suppose the calculation must run now once

Reply:
Yes, didn't think of that. I will add it back.